samza-core/src/main/java/org/apache/samza/metrics/reporter/MetricsHeader.java

```diff
@@ -0,0 +1,160 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.samza.metrics.reporter;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Objects;
+
+
+public class MetricsHeader {
+  private static final String JOB_NAME = "job-name";
+  private static final String JOB_ID = "job-id";
+  private static final String CONTAINER_NAME = "container-name";
+  private static final String EXEC_ENV_CONTAINER_ID = "exec-env-container-id";
+  private static final String SOURCE = "source";
+  private static final String VERSION = "version";
+  private static final String SAMZA_VERSION = "samza-version";
+  private static final String HOST = "host";
+  private static final String TIME = "time";
+  private static final String RESET_TIME = "reset-time";
```

Comment:
In the scala world, these used to be long, will changing it here require any change in the serde to make sure the output on the wire remains the same?